assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -103,31 +154,63 @@ export default function ModuleRecoveryAlert( { id, Notification } ) {
 						/>
 					)
 				}
-				actions={
-					! isLoading &&
-					( hasUserRecoverableModules ? (
-						<RecoverableActions
-							id={ id }
-							recoverableModules={ recoverableModules }
-							userRecoverableModuleSlugs={
-								userRecoverableModuleSlugs
+				learnMoreLink={
+					! isLoading
+						? {
+								label: __( 'Learn more', 'google-site-kit' ),
+								href: documentationURL,
+						  }
+						: undefined
+				}
+				additionalDescription={
+					! isLoading && (
+						<AdditionalDescription
+							selectedModuleSlugs={ selectedModuleSlugs }
+							hasUserRecoverableModules={
+								hasUserRecoverableModules
 							}
 							hasMultipleRecoverableModules={
 								hasMultipleRecoverableModules
 							}
-						/>
-					) : (
-						<UnrecoverableActions
-							id={ id }
 							recoverableModules={ recoverableModules }
 							userRecoverableModuleSlugs={
 								userRecoverableModuleSlugs
 							}
-							hasMultipleRecoverableModules={
-								hasMultipleRecoverableModules
-							}
+							inProgress={ inProgress }
+							setSelectedModuleSlugs={ setSelectedModuleSlugs }
 						/>
-					) )
+					)
+				}
+				ctaButton={
+					hideCTAButton
+						? undefined
+						: {
+								label: __( 'Recover', 'google-site-kit' ),
+								onClick: handleRecoverModules,
+								inProgress,
+								disabled: disableCTA,
+						  }
+				}
+				dismissButton={
+					isLoading
+						? undefined
+						: {
+								label: __(
+									'Remind me later',
+									'google-site-kit'
+								),
+								tertiary: ! hideCTAButton,
+								className: hideCTAButton
+									? 'googlesitekit-banner__cta'
+									: undefined,
```

Comment:
This might be a bit nicer to read/more idiomatic, but I'm not _too_ particular about it 😄 
```suggestion
								className: classnames( { 'googlesitekit-banner__cta': hideCTAButton } )
```